packages/flutter/test/material/popup_menu_theme_test.dart

```diff
@@ -0,0 +1,295 @@
+      ).last,
+    );
+    expect(text.style, popupMenuTheme.textStyle);
+  });
+
+  testWidgets('PopupMenuEntry widget properties take priority over theme',
+          (WidgetTester tester) async {
+    final PopupMenuThemeData popupMenuTheme = _popupMenuTheme();
+    final Key popupButtonKey = UniqueKey();
+    final Key popupButtonApp = UniqueKey();
+    final Key popupItemKey = UniqueKey();
+
+    const Color color = Colors.purple;
+    const ShapeBorder shape = RoundedRectangleBorder(
+      borderRadius: BorderRadius.all(Radius.circular(9.0)),
+    );
+    const double elevation = 7.0;
+    const TextStyle textStyle = TextStyle(
+        color: Color(0x00000000), textBaseline: TextBaseline.alphabetic);
+
+    await tester.pumpWidget(MaterialApp(
+      theme: ThemeData(popupMenuTheme: popupMenuTheme),
+      key: popupButtonApp,
+      home: Material(
+        child: Column(
+          children: <Widget>[
+            PopupMenuButton<void>(
+              key: popupButtonKey,
+              elevation: elevation,
+              color: color,
+              shape: shape,
+              itemBuilder: (BuildContext context) {
+                final List<PopupMenuEntry<Object>> list = <
+                    PopupMenuEntry<Object>>[
+                      PopupMenuItem<void>(
+                        key: popupItemKey,
+                        textStyle: textStyle,
+                        child: const Text(''),
+                      ),
+                ];
+                return list;
+                },
+            ),
+          ],
+        ),
+      ),
+    ));
+
+    await tester.tap(find.byKey(popupButtonKey));
+    await tester.pumpAndSettle();
+
+    final Material button = tester.widget<Material>(
+      find.descendant(
+        of: find.byKey(popupButtonApp),
+        matching: find.byType(Material),
+      ).last,
+    );
+    expect(button.color, color);
+    expect(button.shape, shape);
+    expect(button.elevation, elevation);
+
+    final AnimatedDefaultTextStyle text = tester.widget<AnimatedDefaultTextStyle>(
+      find.descendant(
+        of: find.byKey(popupItemKey),
+        matching: find.byType(AnimatedDefaultTextStyle),
+      ).last,
+    );
+    expect(text.style, textStyle);
+  });
+
+  testWidgets('ThemeData.popupMenuTheme properties are utilized', (
+      WidgetTester tester) async {
+    final Key popupButtonKey = UniqueKey();
+    final Key popupButtonApp = UniqueKey();
+    final Key popupItemKey = UniqueKey();
+
+    await tester.pumpWidget(MaterialApp(
+      key: popupButtonApp,
+      home: Material(
+        child: Column(
+          children: <Widget>[
+            PopupMenuTheme(
+              color: Colors.pink,
+              shape: BeveledRectangleBorder(borderRadius: BorderRadius.circular(10)),
+              elevation: 6.0,
+              textStyle: const TextStyle(color: Color(0xfffff000), textBaseline: TextBaseline.alphabetic),
+              child: PopupMenuButton<void>(
```

Comment:
Remove `<void>`

Reply:
The analyzer says that we can't remove `<void>` because [type annotations must be specified](https://dart-lang.github.io/linter/lints/always_specify_types.html).